### DISCUSSION TITLE
Example sentence for 染まる seems to be mistagged

### DISCUSSION THREAD

**Discussion by Morgawr** [Jitendex/Jitendex#53] [Errors and Corrections]
For the first meaning definition of 染まる: "to be dyed"

The sentence shown is:

JP: 悪習に染まるのは簡単だ。

EN: It's easy to fall into bad habits.

The second meaning definition is: "to be tainted (etc)"

I think this sentence should be keyed on the second meaning instead, as it's closer to that rather than the literal meaning of to be "dyed".

**Comment by stephenmk** [comment]
Thanks, you're correct. I updated the sentence index to use the second sense of 染まる.
